tests/e2e/java_test.go

```diff
@@ -94,7 +155,12 @@ var _ = Describe("odoJavaE2e", func() {
 		It("Should be able to deploy a spring boot uberjar file using openjdk", func() {
 			importOpenJDKImage()
 
-			runCmd("odo create openjdk18 sb-jar-test --binary " + javaFiles + "/openjdk/sb.jar")
+			cmpCreateLog := runCmd("odo create openjdk18 sb-jar-test --binary " + javaFiles + "/openjdk/sb.jar -w")
+			Expect(cmpCreateLog).ShouldNot(ContainSubstring("This may take few moments to be ready"))
+
+			dcName := runCmd("oc get dc | grep sb-jar-test| cut -f 1 -d ' '")
+			Expect(dcName).To(ContainSubstring("sb-jar-test"))
```

Comment:
Check here that the build status is complete

Reply:
In this case (--binary) there is no build

Reply:
Oh OK